spec/grape/validations/validators/coerce_spec.rb

```diff
@@ -525,6 +525,43 @@ def self.parsed?(value)
         expect(last_response.body).to eq('3')
       end
 
+      it 'allows coerce_with to return nil' do
```

Comment:
It would be nice to refactor all those `get`'s below in their own test. Minor.

Reply:
Updated. Although many other tests are written in a similar way here.

Reply:
Yeah and that sucks. Did you refactor all of them? :) Don't hold back.